tests/samples/expected_values.py

```diff
@@ -212,36 +198,29 @@
     "PV2 Current": 0,
     "PV1 Power": 2678,
     "PV2 Power": 0,
-    "Total PV Energy": 8378.7,
-    "Total PV Energy Resets": 1,
+    "Total PV Energy": 8378.8,
     "Today's PV Energy": 26,
     "Grid Frequency Phase 1": 0,
     "Grid Frequency Phase 2": 0,
     "Grid Frequency Phase 3": 0,
-    "Total Energy": 7387.4,
-    "Total Energy Resets": 1,
+    "Total Energy": 7387.5,
     "Today's Energy": 17.8,
     "Battery Voltage": 228.6,
-    "Battery Current": -0.99,
-    "Battery Power": -241,
+    "Battery Current": -1.00,
+    "Battery Power": -242,
     "Battery Temperature": 27,
     "Battery Remaining Capacity": 98,
     "Total Battery Discharge Energy": 2124,
-    "Total Battery Discharge Energy Resets": 0,
     "Today's Battery Discharge Energy": 6.1,
     "Battery Remaining Energy": 12.4,
     "Total Battery Charge Energy": 2501.1,
-    "Total Battery Charge Energy Resets": 0,
     "Today's Battery Charge Energy": 11.9,
     "Exported Power": 0,
-    "Total Feed-in Energy": 3174.79,
-    "Total Feed-in Energy Resets": 4,
-    "Total Consumption": 844.57,
-    "Total Consumption Resets": 1,
+    "Total Feed-in Energy": 3174.83,
```

Comment:
hmm, i would have expected the `Total Feed-in Energy` to increase substantially here. perhaps the u16s are not being combined properly?

Reply:
Ok, all checks out from my side. It seems we were using 65535 instead of 65536 for the multiplier